File: ViewModel/Types/NotificationObject.vb
Imports System.ComponentModel
Imports System.Runtime.CompilerServices

Namespace Types

    Public Class NotificationObject
        Implements INotifyPropertyChanged

        Public Event PropertyChanged(sender As Object, e As PropertyChangedEventArgs) Implements INotifyPropertyChanged.PropertyChanged

        Protected Overridable Sub OnPropertyChanged(<CallerMemberName> Optional propertyName As String = "")
            RaiseEvent PropertyChanged(Me, New PropertyChangedEventArgs(propertyName))
        End Sub

    End Class

End Namespace

File: POFF.Kicker/View/NotificationObject.cs
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace POFF.Kicker.View;

public class NotificationObject : INotifyPropertyChanged
{

    public event PropertyChangedEventHandler PropertyChanged;

    protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = "")
    {
        PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
    }

}